.github/workflows/auto-label-prs.yml

```diff
@@ -0,0 +1,21 @@
+name: Label PRs
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  label-new-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label drafts
+        uses: andymckay/labeler@master
+        if: github.event.pull_request.draft == true
+        with:
+          add-labels: "A3-inprogress"
+          remove-labels: "A0-pleasereview"
+      - name: Label PRs
+        uses: andymckay/labeler@master
+        if: github.event.pull_request.draft == false && ! contains(github.event.pull_request.labels.*.name, 'A2-insubstantial')
```

Comment:
so this enforces that all in progress prs must be drafts, correct?

Reply:
No, it means that this step runs only if it is not a draft (cause we want to set the labels only once the PR is no more WIP)

Reply:
ok thx